### PR TITLE
WIP try rajasegar/ember-cli-update-codemods-manifest/ember-angle-brackets-codemod

### PR DESCRIPTION
**Pull request by kellyselden**
test https://github.com/ember-cli/ember-cli-update-codemods-manifest/pull/16

### CODE REVIEW

**Comment by kellyselden** [comment]
https://github.com/ember-codemods/ember-angle-brackets-codemod/issues/160

**Comment by kellyselden** [comment]
```
      + expected - actual

           "styles": {
             "app.css": ""
           }
           "templates": {
      -      "application.hbs": "{{!-- The following component displays Ember's default welcome message. --}}\r\n<WelcomePage />\r\n{{!-- Feel free to remove this! --}}\r\n\r\n{{outlet}}\r\n\r\n<Ui::Button @text=\"Click me\" />"
      +      "application.hbs": "{{! The following component displays Ember's default welcome message. }}\r\n<WelcomePage />\r\n{{! Feel free to remove this! }}\r\n{{outlet}}\r\n<Ui::Button @text=\"Click me\" />"
             "components": {
               ".gitkeep": ""
             }
           }
```

File: test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{{!-- The following component displays Ember's default welcome message. --}}
+{{! The following component displays Ember's default welcome message. }}
```
Comment (rwjblue) [inline test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs]: Why are the `--` removed here? The original (assuming that `test/fixtures/codemod/local/my-app/app/templates/application.hbs` represents the input) uses `{{!--`, so I would expect the output to continue to use them.

File: test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 <WelcomePage />
-{{!-- Feel free to remove this! --}}
-
-{{outlet}}
\ No newline at end of file
+{{! Feel free to remove this! }}
+{{outlet}}
```
Comment (rwjblue) [inline test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs]: There should be a newline both just before and just after `{{outlet}}` (based on the input template in `test/fixtures/codemod/local/my-app/app/templates/application.hbs`)

**Comment by kellyselden** [comment]
Moved to https://github.com/ember-cli/ember-cli-update-codemods-manifest/pull/16

File: test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs (hunk 1 above)
Comment (kellyselden) [reply test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs]: These were remnants from when the codemod did a lot more formatting, and the expected was altered to match. This has been reverted now.

File: test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs (hunk 2 above)
Comment (kellyselden) [reply test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs]: These were remnants from when the codemod did a lot more formatting, and the expected was altered to match. This has been reverted now.

**Comment by kellyselden** [comment]
Line endings used to be a problem, but aren't anymore. Thanks to everyone involved!

**Pull request closed without merging**